test/unit/routers/alpha-router/functions/get-candidate-pools.test.ts

```diff
@@ -229,4 +261,13 @@ describe('get candidate pools', () => {
       ], { blockNumber: undefined })
     ).toBeTruthy();
   });
+
+  test('getMixedCrossLiquidityCandidatePools', async () => {
+    await getMixedCrossLiquidityCandidatePools({
+      tokenIn: WRAPPED_NATIVE_CURRENCY[1]!,
+      tokenOut: DAI,
+      v2SubgraphProvider: mockV2SubgraphProvider,
+      v3SubgraphProvider: mockV3SubgraphProvider
+    });
```

Comment:
TODO:
mock v2CandidatePools
mock v3CandidatePools

write unit tests for the different scenarios.